src/components/Filter/OptionDisplay.module.css

```
@@ -32,26 +32,25 @@
 }
 
 .optionCheckbox:hover > input {
-  box-shadow: inset 0 0 0 2px var(--fds-semantic-border-input-hover);
+  box-shadow: inset 0 0 0 2px var(--ds-color-accent-border-default);
 }
 
 .option:hover {
-  background-color: var(--fds-semantic-surface-info-subtle);
-  border-bottom: 2px solid var(--fds-semantic-border-input-hover);
+  background-color: var(--ds-global-blue-2);
+  border-bottom: 2px solid var(--ds-color-accent-border-default);
   padding: 4px 0 3px 0;
```

For the same reason as above
```suggestion
  padding: 8px 0 7px 0;
```